ebsco_adapter/ebsco_adapter_iceberg/src/transformers/ebsco_to_weco.py

```diff
@@ -0,0 +1,56 @@
+from pymarc.record import Record
+
+from models.work import SourceIdentifier, SourceWork
+from transformers.alternative_titles import extract_alternative_titles
+from transformers.common import mandatory_field
+from transformers.contributors import extract_contributors
+from transformers.current_frequency import extract_current_frequency
+from transformers.description import extract_description
+from transformers.designation import extract_designation
+from transformers.edition import extract_edition
+from transformers.format import extract_format
+from transformers.genres import extract_genres
+from transformers.holdings import extract_holdings
+from transformers.language import extract_languages
+from transformers.other_identifiers import extract_other_identifiers
+from transformers.production import extract_production
+from transformers.subjects import extract_subjects
+from transformers.title import extract_title
+
+EBSCO_IDENTIFIER_TYPE = "ebsco-alt-lookup"
+
+
+def ebsco_source_identifier(id_value: str) -> SourceIdentifier:
+    return SourceIdentifier(
+        identifier_type="ebsco-alt-lookup", ontology_type="Work", value=id_value
```

Comment:
I think this should use the constant above?
```suggestion
        identifier_type=EBSCO_IDENTIFIER_TYPE ontology_type="Work", value=id_value
```